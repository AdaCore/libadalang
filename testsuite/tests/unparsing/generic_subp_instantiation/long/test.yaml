driver: unparser
rule: generic_instantiation
description: |
  Test line breaks after the `new` keyword and the continuation line
  indentation for `f_generic_subp_name`.
