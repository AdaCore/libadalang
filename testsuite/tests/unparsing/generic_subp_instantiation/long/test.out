procedure Foo is new
  Baaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaar;
